app/src/androidTest/java/mockLocationUtils/MockLocationProvider.java

```diff
@@ -0,0 +1,163 @@
+package mockLocationUtils;
```

Comment:
try using kotlin. it reduces the everyday bugs

Reply:
We'll port mock location class to Kotlin once have some tests for GPS SDK.